### DISCUSSION TITLE
lvm installation problem

### DISCUSSION THREAD

**Discussion by 0xfrossi** [SpiralLinux/SpiralLinux-project#417] [Q&A]
Hey,
I'm trying to install spiral xfce with a fresh LVM partition (testing on VM). After creating it  (manual partitioning) I click on "New volume group" and I select the physical volume. I try with any digits in "pyhsical extent size" but after clicking "Ok" the installer's crashes

Update: I tried debian ISO "debian-live-12.5.0-amd64-xfce" and I had the same problem, so I tried "debian-12.5.0-amd64-netinst" that uses a different installer and no problem here. Well, it seems that the issue is related to the specific installer and not to the Spiral configuration.

![tt](https://github.com/SpiralLinux/SpiralLinux-project/assets/44616736/3f579bbc-6c01-438b-b152-b6741bf8680c)

**Comment by SiriusBlack-8** [comment]
I'm also having a similar problem. I initially tried to create an LVM from the calamari installer too, but it wouldn't handle it. So instead I did a standard install and created the LVM after the fact via LVM2 tools, and the terminal window. I think I did the actual partitioning via GParted. The problem I'm having now, is for some reason the OS won't mount the LVM as the /home directory. Once I figure it out, I'll try to share it with you. Best of luck!

**Reply by geckolinux** [reply]
Hi there, just FYI, Calamares doesn't really support LVM:
https://github.com/calamares/calamares/issues/2343
